stable/cluster-autoscaler/templates/NOTES.txt

```diff
@@ -1,3 +1,18 @@
-To verify that aws-cluster-autoscaler has started, run:
+{{- if or .Values.autoDiscovery.clusterName .Values.autoscalingGroups -}}
+
+To verify that cluster-autoscaler has started, run:
 
   kubectl --namespace={{ .Release.Namespace }} get pods -l "app={{ template "cluster-autoscaler.name" . }},release={{ .Release.Name }}"
+
+{{- else -}}
+
+##############################################################################
+####   ERROR: You must specify values for either                          ####
+####   autoDiscovery.clusterName or autoscalingGroups[]                   ####
+##############################################################################
+
+The deployment and pod will not be created and the installation is not functional
```

Comment:
is there an example `helm upgrade` we could put there? e.g. `helm upgrade {{.Release.Name}} --set autoDiscovery.clusterName=YOUR_CLUSTER_NAME`

Reply:
@prydonius I could add something for autoscalingGroups. Autodiscovery requires setting tags on the asg. It's unlikely that either will work on a default Kops install without IAM changes. I figured the best thing to do was point them to the readme rather than try to duplicate that documentation.

Reply:
@afirth makes sense, thanks 👍